Backend/ActiveCruzer/Start/Startup.cs

```diff
@@ -36,8 +40,15 @@ public Startup(IWebHostEnvironment env, IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            // add service from mysql framework
             services.AddDbContext<ACDatabaseContext>(options =>
-                options.UseSqlServer(_configuration.GetConnectionString("ActiveCrzuerDB-ConnectionString")));
+                options.UseMySql(_configuration.GetValue<string>("ActiveCrzuerDB-ConnectionString")));
+
+            services.AddMvc();
```

Comment:
I think this must be the last statement  in the "Configure service"

Reply:
placed it at the end